.github/workflows/build.yml

```diff
@@ -252,7 +252,7 @@ jobs:
     - name: Display build info
       run: make pythoninfo
     - name: Tests
-      run: make buildbottest TESTOPTS="-j4 -uall,-cpu"
```

Comment:
was it okay to remove -j4?  i believe CI gives us a single core two vcpu thread instance so 4 seems reasonable.  is that what regrtest defaults to?  anyways, given this PR is in - pay attention to CI execution times and see how they compare afterwards.

Reply:
GHA announces 2 CPUs, regrtest uses 4 processes: it's the same as -j4. If later, we notice that sometimes goes wrong, obivously, we can adjust the -jN value. regrtest uses "2 + ncpu" worker processes.

Example on the GHA Ubuntu job.

* test.pythoninfo announce ``os.cpu_count: 2``
* ``Run 463 tests in parallel using 4 worker processes (timeout: 10 min, worker timeout: 15 min)``

Before when -j4 was used: https://github.com/python/cpython/actions/runs/6305743468/job/17119712665

```
0:00:00 load avg: 1.20 Run 463 tests in parallel using 4 worker processes (timeout: 20 min, worker timeout: 25 min)
```

The same number of worker processes is used before and after.